### PR TITLE
feat: Improve Open Source Page 

### LINKED ISSUE #165: [Feature Request] Improve Open Source Page Design
### Overview

Improve the design of the open source page to make it look more appealing.

### Details

Make it look like attached screenshot.

### Why would this feature be useful?

Adding the open source drawing and improving the layout would increase the visual appeal.

### Additional Notes

Mockup:

The image to use:

### PR DESCRIPTION
**Pull request by codingzilin**
### Description
Improve open source page.

### Changes Made
Added title section, introduction section, photo and replaced button with hyperlink.

### Related Issues
Resolve #165

### Additional Notes
image

### CODE REVIEW

File: public/images/opensource/csclub_open_source-2.png
Comment (phoenixpereira) [inline public/images/opensource/csclub_open_source-2.png:1]: Change the file name to just `opensource.png`

File: src/app/open-source/page.tsx [hunk 1]
```diff
@@ -19,24 +22,73 @@ export default function OpenSourcePage() {
...
+                                        alt="Yellow Triangle"
+                                        className=" mr-4"
+                                        width={40}
+                                        height={40}
+                                    />
+                                    <Link
+                                        className="text-lg font-bold underline md:text-3xl"
+                                        href={
+                                            'https://docs.google.com/forms/d/e/1FAIpQLSe2uvnn4qW95yJ7TzyDo7QMjzhkawvdERRDmeBLMwloi-nqjg/viewform'
+                                        }
```
Comment (jsun969) [inline src/app/open-source/page.tsx:72]:
```suggestion
                                        href="https://docs.google.com/forms/d/e/1FAIpQLSe2uvnn4qW95yJ7TzyDo7QMjzhkawvdERRDmeBLMwloi-nqjg/viewform"
```

File: src/app/open-source/page.tsx [hunk 2]
```diff
@@ -19,24 +22,73 @@ export default function OpenSourcePage() {
...
+                                        build innovative open-source software together. We are a new
+                                        team currently working on a few projects, but we have many
+                                        more exciting projects planned for the future!
+                                    </p>
+                                </div>
+                                <div className="flex flex-row">
+                                    <Image
+                                        src="/images/yellow-triangle.svg"
+                                        alt="Yellow Triangle"
+                                        className=" mr-4"
```
Comment (jsun969) [inline src/app/open-source/page.tsx:64]:
```suggestion
                                        className="mr-4"
```

File: src/app/open-source/page.tsx [hunk 3]
```diff
@@ -19,24 +22,73 @@ export default function OpenSourcePage() {
...
+                                    >
+                                        Join the Open Source Team!
+                                    </Link>
+                                </div>
+                            </div>
+                            <div className="relative z-0 mt-8 md:w-auto">
+                                <FancyRectangle colour={'white'} offset={'20'} filled rounded>
+                                    <Image
+                                        src={'/images/opensource/csclub_open_source-2.png'}
+                                        alt={'Open source'}
```
Comment (jsun969) [inline src/app/open-source/page.tsx:82]:
```suggestion
                                        src="/images/opensource/csclub_open_source-2.png"
                                        alt="Open source"
```

**Review by jsun969 — CHANGES_REQUESTED** [review]

File: src/app/open-source/page.tsx [hunk 4]
```diff
@@ -19,24 +22,73 @@ export default function OpenSourcePage() {
...
+                                    </div>
+                                    <div className="relative flex flex-row">
+                                        <div className="w-12 bg-orange" />
+                                        <div className="relative flex flex-1 flex-col">
+                                            <span className="relative z-[2] bg-white py-4 pl-4 pr-16 text-2xl font-black text-black lg:pr-36 lg:text-3xl">
+                                                Who is the
+                                            </span>
+                                            <span className="relative z-[2]  whitespace-nowrap border-2 border-white py-6 pl-4 pr-16 text-2xl underline lg:pr-36 lg:text-3xl">
+                                                Open Source Team?
+                                            </span>
```
Comment (phoenixpereira) [inline src/app/open-source/page.tsx:43]: Make this bold.

**Review by phoenixpereira — CHANGES_REQUESTED** [review]

File: src/app/open-source/page.tsx [hunk 5]
```diff
@@ -19,23 +22,68 @@ export default function OpenSourcePage() {
...
+                                        <Duck colour="white" size={40} />
+                                        <Duck colour="white" size={40} />
+                                    </div>
+                                    <div className="relative flex flex-row">
+                                        <div className="w-12 bg-orange" />
+                                        <div className="relative flex flex-1 flex-col">
+                                            <span className="relative z-[2] bg-white py-4 pl-4 pr-16 text-2xl font-black text-black lg:pr-36 lg:text-3xl">
+                                                Who is the
+                                            </span>
+                                            <span className="relative z-[2]  whitespace-nowrap border-2 border-white py-6 pl-4 pr-16 text-2xl font-bold underline lg:pr-36 lg:text-3xl">
```
Comment (jsun969) [inline src/app/open-source/page.tsx:41]:
```suggestion
                                            <span className="relative z-[2] whitespace-nowrap border-2 border-white py-6 pl-4 pr-16 text-2xl font-bold underline lg:pr-36 lg:text-3xl">
```

**Review by jsun969 — CHANGES_REQUESTED** [review]

**Review by jsun969 — APPROVED** [review]

File: src/app/open-source/page.tsx [hunk 6]
```diff
@@ -19,23 +22,68 @@ export default function OpenSourcePage() {
...
-                    <section className="mb-8 flex justify-center text-black">
-                        <Button
-                            type="button"
-                            colour="orange"
-                            width="w-full"
-                            href="https://docs.google.com/forms/d/e/1FAIpQLSe2uvnn4qW95yJ7TzyDo7QMjzhkawvdERRDmeBLMwloi-nqjg/viewform"
-                        >
-                            Join Us Now
-                        </Button>
+                        <div className="grid grid-cols-1 gap-8 lg:grid-cols-2">
```
Comment (phoenixpereira) [inline src/app/open-source/page.tsx:25]: Fix the overlap at about 3/4 screen width by making the image go below the "Join the Open Source Team!" earlier (between `xl` and `2xl`). 

image

File: src/app/open-source/page.tsx [hunk 7]
```diff
@@ -19,23 +22,68 @@ export default function OpenSourcePage() {
...
+                                        className="mr-4"
+                                        width={40}
+                                        height={40}
+                                    />
+                                    <Link
+                                        className="text-lg font-bold underline md:text-3xl"
+                                        href="https://docs.google.com/forms/d/e/1FAIpQLSe2uvnn4qW95yJ7TzyDo7QMjzhkawvdERRDmeBLMwloi-nqjg/viewform"
+                                    >
+                                        Join the Open Source Team!
+                                    </Link>
```
Comment (phoenixpereira) [inline src/app/open-source/page.tsx:72]: Make the text a bit bigger for `sm`-`md` widths.

```suggestion
                                    <Link
                                        className="text-lg font-bold underline sm:text-2xl md:text-3xl"
                                        href="https://docs.google.com/forms/d/e/1FAIpQLSe2uvnn4qW95yJ7TzyDo7QMjzhkawvdERRDmeBLMwloi-nqjg/viewform"
                                    >
                                        Join the Open Source Team!
                                    </Link>
```

File: src/app/open-source/page.tsx
Comment (phoenixpereira) [inline src/app/open-source/page.tsx:1]: Can you also update the text at the bottom to match this? 
Austin's GitHub profile link is: https://github.com/AustinH-adl

image

**Review by phoenixpereira — APPROVED** [review]

**Pull request merged**